src/libraries/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -3453,4 +3453,7 @@ Usage: dotnet {0} [--assembly &lt;assembly file path&gt;] [--type &lt;type name&
   <data name="CompilingScriptsNotSupported" xml:space="preserve">
     <value>Compiling JScript/CSharp scripts is not supported</value>
   </data>
-</root>
+  <data name="XPath_UnexpectedTokens" xml:space="preserve">
```

Comment:
This doesn't match the C# value below. It doesn't break anything as that code is dead. I pushed a change to reverse these and will put up a PR to remove the code.